### ISSUE TITLE
[BUG] Using setBreakpoints() with larger top breaks position in inverse

### ISSUE THREAD

**Issue by akharag** [roman-rr/cupertino-pane#92]
**Describe the bug**

The cupertino pane moves to the wrong position when these conditions occur:
1. The pane is initialized as inverse.
2. The setBreatpoints method changes the top breakpoint to a value larger than when it was initialized.

**To Reproduce**

page.ts
```javascript
this.myPane = new CupertinoPane(
      ".cupertino-pane-main", // Pane container selector
      {
        initialBreak: "top",
        breaks: {
          top: { enabled: true, height: screen.height * 0.65 },
          middle: { enabled: false },
          bottom: { enabled: true, height: screen.height * 0.1 },
        },
        buttonClose: false,
        inverse: true,
      }
    );

  nextSlide() {
    this.slides.slideNext();
    this.mainPane.setBreakpoints({
      top: {
        enabled: true,
        height: screen.height,
        bounce: true,
      },
      middle: { enabled: false },
      bottom: { enabled: true, height: 35 },
    });
  }
```

page.html
```html
<ion-content>
    <div class="cupertino-pane-main">
        <ion-slides>

            <ion-slide>
                content
            <ion-slide>

            <ion-slide>
                content
            <ion-slide>

        </ion-slides>
    </div>
</ion-content>
```

**Expected behavior**

The pane should be anchored at the top of the page after the setBreakpoints method has been called.

**Screenshots**

![Screen Recording 2020-11-18 at 1 23 17 AM](https://user-images.githubusercontent.com/22301549/99464182-2c21f200-2940-11eb-9c42-97e93c3f24d1.gif)

**Desktop (please complete the following information):**
 - Browser [e.g. chrome, safari]: chrome
 - Version [e.g. 22] 86.0.4240.193

**Smartphone (please complete the following information):**

**Additional context**

**Comment by roman-rr** [comment]
@akharag thank you i got it

**Comment by roman-rr** [comment]
@akharag already fixed in latest master commit. Will be included in next release `1.2.0`
